ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -329,6 +329,11 @@ ORK_CLASS_AVAILABLE
  */
 @property (readonly, getter=isVertical) BOOL vertical;
 
+/**
```

Comment:
minor nit - remove (read-only) from comment

Reply:
I was referring to the comment that is part of the PR for hideSelectedValue - "A Boolean value indicating whether the selected value should be hidden. (read-only)"